opentelemetry-exporter-gcp-logging/src/opentelemetry/exporter/cloud_logging/version.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2025 The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__version__ = "0.9b0"
```

Comment:
```suggestion
__version__ = "1.9.0.dev0"
```

we keep everything in lockstep https://github.com/search?q=repo%3AGoogleCloudPlatform%2Fopentelemetry-operations-python+%2F%5E__version__+%3D%2F&type=code